src/csharp/Microsoft.Spark/Microsoft.Spark.csproj

```diff
@@ -14,6 +14,7 @@
 
   <ItemGroup>
     <InternalsVisibleTo Include="Microsoft.Spark.Experimental" />
+    <InternalsVisibleTo Include="Microsoft.Spark.Extensions.Delta" />
```

Comment:
Delta or DeltaLake?

Reply:
We could call it `DeltaLake`.  Their namespace is `io.delta.*`.

Reply:
Thoughts @imback82  @suhsteve  ?

Reply:
I would go with delta since their jar name is also delta, not deltalake.

Reply:
Ok sticking with Delta.